travelpal/assets/js/app.js

```diff
@@ -19,3 +19,12 @@ import "phoenix_html"
 // paths "./socket" or full ones "web/static/js/socket".
 
 // import socket from "./socket"
+import store from './store';
+import api from './api';
+import travelpal_init from "./components/travelpal";
+
+$(function() {
+  api.request_users();
+  api.request_friends();
+  travelpal_init(store);
+});
```

Comment:
Does this mean we call the api even if the user is not logged in? Since this is before the travelPal_init code?

Reply:
Yes, it is possible. Those two API requests are just querying the db for everything in the users and friends tables. We might move those requests later.

Reply:
👍